tasks/quality_control/task_screen.wdl

```diff
@@ -115,15 +114,18 @@ task check_reads {
             flag="FAIL; the estimated coverage is less than the minimum of ~{min_coverage}x"
           else
             flag="PASS"
+            echo $estimated_genome_size | tee EST_GENOME_LENGTH
           fi 
         fi
       fi 
     fi 
     
     echo $flag | tee FLAG
+    echo $estimated_genome_size | tee EST_GENOME_LENGTH
   >>>
   output {
     String read_screen = read_string("FLAG")
+    Int? est_genome_length = read_string("EST_GENOME_LENGTH")
```

Comment:
[read_int()](https://github.com/openwdl/wdl/blob/main/versions/1.0/SPEC.md#int-read_intstringfile) may be more appropriate to capture estimated_genome_length. Also, since this is set as 0 on line 117, does it need to be set as an optional output (_i.e._ `Int` v. `Int?`)?

Reply:
Fixed this!